src/ontology/Makefile

```diff
@@ -216,6 +216,11 @@ VQUERIES = $(foreach V,$(VCHECKS),$(SPARQLDIR)/$V-violation.sparql)
 sparql_test: $(SRC)
 	$(ROBOT) verify -i $< --queries $(VQUERIES) -O reports/
 
+ensure_terms_%: %
+	$(ROBOT) query -i $< --query ../sparql/ensure-terms-$<.sparql ensure-terms-$<.txt && grep -q true ensure-terms-$<.txt
+
+test: ensure_terms_hpo_slim.owl
```

Comment:
There is already a test rule in line 35. Should they be merged?

Reply:
Not necessary in make: this is simply extending the respective make target `test`, so it i=ensures that the ensure_terms test is also run when when `test` is run. I changed this now to `all` to ensure that the test is run always run when a release is run!

Reply:
Oh, I see. Thanks for clarifying.